pkg/config/resources.go

```diff
@@ -28,9 +31,11 @@ func (rn ResourceName) String() string {
 const (
 	CSIController                         ResourceName = "csi-controller"
 	CSINode                               ResourceName = "csi-node"
+	CSIHostDefiner                        ResourceName = "csi-hostdefiner"
```

Comment:
why do we have `csi` as a prefix?

Reply:
wanted that it will be like `CSINode` and controller, do you think it better without the CSI?

Reply:
its not csi..

Reply:
done

Reply:
I still see e.g. `csi_hostdefiner.go`